### PR TITLE
Scraping reading room links

### PR DESCRIPTION
**Pull request by khandelwal**
This is a first pass at scraping reading room links from the various FOIA websites in an attempt to build that into the dataset. There's a new script to scrape and update the reading room links, and then the changes themselves. 

Each agency/department might have multiple reading room links (for a variety of reasons).

### CODE REVIEW

File: contacts/data/ABMC.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ departments:
   name: American Battle Monuments Commission
   phone: 703-696-6897
   public_liaison: 'Michael G. Conley, Phone: (703) 696-6778'
+  reading_rooms:
+  - !!python/tuple
```
Comment (jackiekazil) [inline contacts/data/ABMC.yaml]: From my understanding this returns the data in tuple, but this feels a little like coding in data. Data formats like YAML should be language agnostic. What is the gain by adding this that you don't get from not having it?

Comment (khandelwal) [inline contacts/data/ABMC.yaml]: That's how the yaml parser writes out the list of tuple (I didn't explicitly add that in).  But you bring up a good point, I should probably ensure the YAML generated is language agnostic.

**Comment by konklone** [comment]
Is this still in progress?

**Comment by khandelwal** [comment]
This is ready for review.

**Comment by rjmajma** [comment]
Pop Pop!

**Comment by khandelwal** [comment]
@ramirezg or @konklone Can either of you review this?

**Comment by konklone** [comment]
This is awesome, and a well done job of scraping + doing manual review.

My one complaint is I think the data format works better as an array of objects, rather than an array of arrays. In other words, what's there now is:

``` yaml
  reading_rooms:
  - - FOIA Reading Room
    - http://www.abmc.gov/node/1005/foia-reading-room
```

Which yields `[[name, url], [name, url]...`.

I think it would be less brittle and more clear as:

``` yaml
  reading_rooms:
  - name: FOIA Reading Room
    url: http://www.abmc.gov/node/1005/foia-reading-room
```

Which yields `[{name: name, url: url}, {name: name, url: url}, ...]`.

**Pull request merged**